XamlControlsGallery/ControlPages/ButtonPage.xaml

```diff
@@ -18,13 +18,12 @@
     <StackPanel>
         <local:ControlExample x:Name="Example1" HeaderText="A simple Button with text content.">
             <local:ControlExample.Example>
-                <RelativePanel>
-                    <Button x:Name="Button1" Content="Standard XAML button" Click="Button_Click"
-                            IsEnabled="{x:Bind DisableButton1.IsChecked.Value.Equals(x:False), Mode=OneWay}" />
-                    <TextBlock x:Name="Control1Output" Margin="8,0,0,0" FontFamily="Global User Interface"
-                            RelativePanel.RightOf="Button1" RelativePanel.AlignVerticalCenterWith="Button1" />
-                </RelativePanel>
+                <Button x:Name="Button1" Content="Standard XAML button" Click="Button_Click"
+                    IsEnabled="{x:Bind DisableButton1.IsChecked.Value.Equals(x:False), Mode=OneWay}" />
```

Comment:
Wait, we can do cool things like this in markup? ❤
🌈The more you know 🌈

Reply:
😀